style.css

```diff
@@ -14,6 +14,239 @@ nav {
   left: 25%;
 }
 
+.works1 {
+  background-color: #dfe1e6;
+  display: grid;
+  grid-template-columns: 100%;
+  grid-auto-rows: minmax(552px, auto);
+  grid-gap: 10px;
+  width: 100%;
+}
+
+.worksec1 {
+  position: static;
+  width: 88%;
+  height: 310px;
+  margin: 25px;
+  left: 50%;
+  top: 40%;
+  border-radius: 9px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  background: white;
+  padding-bottom: 142px;
+  padding-top: 76px;
+}
+
+.cardpic {
+  margin: 25px;
+  margin-top: -40px;
+}
+
+.project1-btn {
+  position: static;
+  width: 110px;
+  height: 40px;
+  font-style: normal;
+  border-radius: 6px;
+  font-weight: 380;
+  font-size: 17px;
```

Comment:
- [ ] Great job implementing this class 👍. However, the button and list items on your site have a different design than that of Figma's.
Your Implementation | Figma Design
:---: | :---:
![image](https://user-images.githubusercontent.com/59416136/153672275-394b840e-70b4-44e1-acb8-875c8ff10c59.png)  |  ![image](https://user-images.githubusercontent.com/59416136/153672368-940715d9-db3a-4275-bb6f-0cc5e86cae38.png)

Kindly fix this.